src/plugins/index_pattern_management/public/components/edit_index_pattern/edit_index_pattern.tsx

```diff
@@ -121,7 +121,7 @@ export const EditIndexPattern = withRouter(
     const refreshFields = () => {
       overlays.openConfirm(confirmMessage, confirmModalOptionsRefresh).then(async (isConfirmed) => {
         if (isConfirmed) {
-          await indexPattern.init(true);
+          await indexPattern.refreshFields();
```

Comment:
how will this affect user experience ?

Reply:
Calling `init` will load the index pattern fresh but `save` has functionality to apply the updated fieldlist to an up to date indexPattern if it fails on first try. Practically speaking, nothing.